lib/tasks/detailed_guides.rake

```diff
@@ -0,0 +1,32 @@
+namespace :detailed_guides do
+  desc "Relocate detailed guides referenced in Topics"
+  task :relocate_to_guidance => :environment do
+    panopticon = CollectionsPublisher.services(:panopticon)
+
+    page = 1
+    while true
```

Comment:
If I use `get_list!` instead of `get_json`, I get this error:

```
NoMethodError: undefined method `grouped_results' for #<Array:0x007f173c1960b8>
/var/govuk/collections-publisher/lib/tasks/detailed_guides.rake:8:in `each'
```

Which I guess means the request I'm making isn't compatible with `get_list!`?